src/gatsby/onCreateNode.ts

```diff
@@ -39,13 +41,20 @@ export default function onCreateNode({
       value: false,
     });
 
-    const markdownNode = {
+    // TODO(epurkhiser): We need to adctually correctly type the ApiEndpoint
+    // node types and discriminate the type to correctly represent fields like
+    // description on the node (otherwise it falls into the Record<string,
+    // unknown> type)
+    const description = node.description as string;
```

Comment:
```suggestion
    // TODO(epurkhiser): We need to actually correctly type the ApiEndpoint
    // node types and discriminate the type to correctly represent fields like
    // description on the node (otherwise it falls into the Record<string,
    // unknown> type)
    const description = node.description as string;
```